### ISSUE TITLE
Add shift fuzzer

### ISSUE THREAD

**Issue by disnet** [sweet-js/sweet.js#554]
Shift recently released a really nice [fuzzer](http://shift-ast.org/fuzzer.html). It would be great to add this as an npm script so we can start tracking down parser bugs.

**Comment by vendethiel** [comment]
Is it actually new? Isn't it @michaelficarra's fuzzer, updated / repurposed?

**Comment by disnet** [comment]
Yep the shift fuzzer is totally new, supports ES2015 (I don't think esfuzz has been updated for 2015), and was written by @bakkot + @michaelficarra.

**Comment by vendethiel** [comment]
Okay, thanks for the correction.

**Comment by gabejohnson** [comment]
@disnet I think this is going to require an upgrade to `shift-*`. It doesn't support `shift-parser` < 4.0

**Comment by gabejohnson** [comment]
To be clear, this is only an issue b/c version mismatch. If we could use a new version of `shift-reducer` just w/ the fuzzer then we would be fine.

**Comment by michaelficarra** [comment]
@gabejohnson With the way npm manages dependencies, is this actually an issue? You should be able to have many side-by-side incompatible installs of shift-reducer.

**Comment by gabejohnson** [comment]
@michaelficarra shift-codegen 4.0.0 is a top-level dependency. shift-fuzzer would requires version >= 5. The error I encountered re shift-parser was due to it being a dependency of shift-codegen (and node picking up v4). It's a moot point as @disnet is migrating to newer versions of shift-*.
